webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -168,17 +170,21 @@ const ModalForm: React.FC<Props> = ({ form, user, groups, viewOnly, roles }) =>
               optionFilterProp="children"
               placeholder={viewOnly ? 'No Roles Added' : 'Add Roles'}
               showSearch>
-              {knownRoles.map((r) => (
-                <Select.Option
-                  disabled={
-                    roles?.find((ro) => ro.id === r.id)?.fromGroup?.length ||
-                    roles?.find((ro) => ro.id === r.id)?.fromWorkspace?.length
-                  }
-                  key={r.id}
-                  value={r.id}>
-                  {r.name}
-                </Select.Option>
-              ))}
+              {Loadable.match(knowRolesLoadable, {
+                Loaded: () =>
+                  knownRoles.map((r) => (
+                    <Select.Option
+                      disabled={
+                        roles?.find((ro) => ro.id === r.id)?.fromGroup?.length ||
+                        roles?.find((ro) => ro.id === r.id)?.fromWorkspace?.length
+                      }
+                      key={r.id}
+                      value={r.id}>
+                      {r.name}
+                    </Select.Option>
+                  )),
+                NotLoaded: () => undefined,
```

Comment:
same here

https://github.com/determined-ai/determined/pull/5522#discussion_r1042317154

We should add loading state (spinner) or `// TODO show spinner when this is loading`

Reply:
@keita-determined  Something might happened to Github at that time, it's working now